### ISSUE TITLE
fix PN glomerulus identification 

### ISSUE THREAD

**Issue by jefferis** [flyconnectome/tracerutils#47]
These now seem to be identified by name rather than annotation. 

Something like this seems to work:

```
pns.fafb[,'glom']=stringr::str_match(catmaid_get_neuronnames(pns.fafb[,'skid']),'glomerulus (\\S+)')[,2]
```

**Comment by jefferis** [comment]
Should add an option to do this to `find_glomeruli()`

**Comment by fmlove** [comment]
Are the annotations no longer being updated?  I would have thought they'd be more useful, particularly with mPNs.

**Comment by schlegelp** [comment]
I use a combination of annotations. E.g. for right canonical DA1 uPNs I would intersect annotations `glomerulus DA1` and `WTPN2017_excitatory_uPN_right`.

**Comment by fmlove** [comment]
That's what I do for uPNs anyway.  What I don't understand is why this broke - shouldn't the annotations still be more robust than names?  I'm not averse to adding a name-based option to `find_glomeruli()`, but I'm not sure on the reasoning.

**Comment by jefferis** [comment]
@fmlove There are a number of glomerulus annotations that are wrong (e.g. glomerulus VC3m/l).

@schlegelp
> I use a combination of annotations. E.g. for right canonical DA1 uPNs I would intersect annotations glomerulus DA1 and WTPN2017_excitatory_uPN_right.

It sounds like this finds RHS PNs that innervate a particular glomerulus but does not tell you which glomerulus a neuron innervates when you are given the skid.

**Comment by jefferis** [comment]
Hi @SridharJagannathan – I think we did some work on this together on your machine. Could you push?

**Comment by SridharJagannathan** [comment]
Hi @jefferis I have pushed the latest changes, can you please check

**Comment by SridharJagannathan** [comment]
Also @jefferis won't it be nicer to add testcases for any additional functionality we implement, I tested the examples offlines but ideally would be nicer to have test cases itself.

**Comment by jefferis** [comment]
Great. You could def add test cases. These could be live against the server conditioned on whether access credentials exist on the current machine (tracerutils is basically specific to those working with adult FAFB).

**Comment by jefferis** [comment]
This is now closed as of 282f8779f412c399239a51d0b6e19d1559b8d82a
